### ISSUE TITLE
Re-opening #985, A32NX doesn't start without ground power

### ISSUE THREAD

**Issue by Flaxx-Rat** [flybywiresim/a32nx#7339]
### Aircraft Version

Experimental

### Build info

```json
{
    "built": "2022-07-16T14:32:40+00:00",
    "ref": "refs/heads/experimental",
    "sha": "47ce3846527a2964adffeda7b60eb5ca200fc240",
    "actor": "tracernz",
    "event_name": "manual",
    "pretty_release_name": "experimental:47ce3846"
}
```

### Describe the bug

1. When placing the plane on a parkung spot of a smaller airfield (e.g. EGPO) having no ground service at all, it is not possible powering the plane on in order to start the APU.
2. Placing the plane on a runway, PFD and MCDU are remaining inoperative, as well as the flight management computer.

### Expected behavior

APU should be available even without external power.
Placing a plane on a runway should cause a full empored plane, ready  to final configuration for take-off

### Steps to reproduce

1. Place a plane on a parkin spot of EGPO
2. Try to start BAT and then APU (Voltage will be shown to be 0 V)

1. Try to place on the active runway
2. The result will be a cockpit that can not be prepared to start.

**Comment by Benjozork** [comment]
Tested both of those things and I cannot reproduce the issue.

Can you check which version of the sim you have ?

**Comment by tracernz** [comment]
Batt voltage = 0 indicates the systems WASM isn't loading/running at all... Please check reported/known issues https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/#autopilot-fadec-electrical-systems-not-working-as-intended-utf8-issue

**Comment by Flaxx-Rat** [comment]
@Benjozork : It's the Premium Deluxe 1.26.5.0, my Win10 is localized to Germany.
@tracernz : Simconnect didn't appaer in my Documents folder. I've switched now to Beta UTF8 in my settings, nonetheless the fault remains (I've switched as well to the stable version 0.8.2 with no avail). 
The pictures are showing the latter after placing the FBW32NX at 
1. EGPO, Parkposition 67
![2022-07-17 09_19,16_AB](https://user-images.githubusercontent.com/109431387/179389372-354128c0-c9f7-4bff-acbb-6aa7ca634bc5.jpg)

2. EGPO|RW26:
![2022-07-17 09_24,58_AB](https://user-images.githubusercontent.com/109431387/179389206-ceef1b53-9582-4534-b49e-cac89e0dbab3.jpg)

**Comment by Flaxx-Rat** [comment]
It had been my mistake. Even if I had tested, before writing here, switching off all possible infuences by other add-ons (I'm using the MSFS-Addons-Linker), I overlooked a quite old mod of '_mugz_' that still was sitting in the Community folder as it had been installed manually there a while ago. Now I feel quite sheepish - sorry for bothering you.

Actually, I became suspicious when I got working batteries after I finally shifted the rest of my community contend away. I thougt about FSUIPC, NAVIGRAPH and PMDG and nothing else.
